cosmos/spring-cloud-azure-starter-data-cosmos/spring-cloud-azure-data-cosmos-aks-sample/src/main/java/spring/cloud/azure/starter/data/cosmos/sample/aks/CosmosProperties.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The 2 files are not deleted yet.
![image](https://user-images.githubusercontent.com/13167207/203196784-30764b7d-a883-44c0-9d87-98ed29c14e7a.png)

Reply:
deleted